nucleus/deploy/client.py

```diff
@@ -346,7 +364,12 @@ def sync_request(
         return resp["result_url"]
 
     def async_request(
-        self, endpoint_id: str, url: str, return_pickled: bool = True
+        self,
+        endpoint_id: str,
+        *,
+        url: Optional[str] = None,
```

Comment:
Adding the kwargs functionality is also done in pr https://github.com/scaleapi/nucleus-python-client/pull/232, guess one of us will have a small merge conflict lol